### ISSUE TITLE
Delete old data from perf machines

### ISSUE THREAD

**Issue by ajcvickers** [aspnet/EntityFramework#8173]
Old data was so out-of-whack that it is making current data hard to read. It should be soft or otherwise deleted.

**Comment by divega** [comment]
I have moved the data from the time period in which there were issues to a separate table using the following:

``` SQL
SELECT  *
INTO BadRuns
FROM dbo.Runs 
WHERE RunStarted BETWEEN '20170124' AND '20170327' 

DELETE 
FROM dbo.Runs 
WHERE RunStarted BETWEEN '20170124' AND '20170327' 
```

A few observations: 
- Whatever the malfunction was that caused the performance dive, it seems that it only affected tests that access the database, e.g. the `DbSet` operation tests were not affected by it.
- Since the bad data has been removed, it is now easier to observe that there has been a recent improvement in the EF6 results. My hypothesis is that this is a consequence of the upgrade of the machine to .NET 4.6.1
- Some tests show a pronounced improvement in EF Core results as well, in particular the update tests. This could be due to a product change.
